### PR TITLE
Added Test case for ContentCardUIUtils

### PR DESCRIPTION
**Pull request by navratan-soni**
## Description

Added Unit Test cases for ContentCardUIUtils

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/messaging/src/test/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaDataUtilsTest.kt [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+    @Test
+    fun `test createAepText with valid content`() {
+        val titleMap = contentCardMap[MessagingConstants.ContentCard.UIKeys.TITLE] as? Map<String, Any>
+        val result = ContentCardSchemaDataUtils.createAepText(titleMap ?: emptyMap())
+        assertEquals("Messaging SDK Smoke Test", result.content)
+    }
+
+    @Test
+    fun `test createAepText with missing content`() {
```
Comment (spoorthipujariadobe) [inline code/messaging/src/test/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaDataUtilsTest.kt:108]: nit: can we also add a `test createAepText with null content` ?

Comment (spoorthipujariadobe) [reply code/messaging/src/test/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaDataUtilsTest.kt:108]: Same suggestion for image (null url) and button (null content, interactionId and actionUrl)

**Review by spoorthipujariadobe — APPROVED** [review]

File: code/messaging/src/test/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaDataUtilsTest.kt [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        Mockito.`when`(proposition.uniqueId).thenReturn("testId")
+
+        contentCardSchemaData.parent = propositionItem
+
+        val result = ContentCardSchemaDataUtils.getTemplate(contentCardSchemaData)
+        assertTrue(result is SmallImageTemplate)
+    }
+
+    @Test
+    fun `test getTemplate with missing title`() {
```
Comment (spoorthipujariadobe) [inline code/messaging/src/test/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaDataUtilsTest.kt:202]: nit: Can we also have a ``test getTemplate with empty title`

Comment (spoorthipujariadobe) [reply code/messaging/src/test/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaDataUtilsTest.kt:202]: We can also add a `test getTemplate with missing metadata map` which should throw an exception
